backend/src/main/java/fr/anettehaferkorn/backend/model/WineGrape.java

```diff
@@ -0,0 +1,22 @@
+package fr.anettehaferkorn.backend.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Document("WineGrapes")
+public class WineGrape {
+
```

Comment:
Why WineGrape and not just Wine?

Reply:
because so far these are only grape recommendations, e.g. Chardonnay, Riesling, Merlot and not explicitly wines in the sense of one of the wines XY from xyz